scripting/get5/pausing.sp

```diff
@@ -2,31 +2,55 @@ public bool Pauseable() {
   return g_GameState >= Get5State_KnifeRound && g_PausingEnabledCvar.BoolValue;
 }
 
+public bool PauseGame(MatchTeam team, Get5PauseType type, int pausesLeft) {
+
```

Comment:
Please remove this extraneous line.

(My opinion is basically that ofhttps://google.github.io/styleguide/cppguide.html#Vertical_Whitespace - this does not aid readability)